components/viaduct/src/fetch_msg_types.proto

```diff
@@ -25,7 +25,6 @@ message Request {
     optional bytes body = 3;
     map<string, string> headers = 4;
     required bool follow_redirects = 5;
-    required bool include_cookies = 6;
```

Comment:
nit: are we able to leave some sort of comment here to explain the jump from `5` to `7` for future readers?

Reply:
Alternately, would it actually be safe for us to re-number these fields? I don't think we have the usual backwards-compatibility concerns with an updated consumer received an older-format protobuf from an outdated producer.

Reply:
Yup that's safe, just didn't want to pollute the git blame, but I guess it doesn't really matter.